scripts/reloader.bash

```diff
@@ -1,15 +1,25 @@
 #!/bin/bash
+BASH_IT_LOG_PREFIX="core: reloader: "
 pushd "${BASH_IT}" >/dev/null || exit 1
 
-# TODO: Add debugging output
+function _set-prefix-based-on-path()
+{
+  filename=$(_bash-it-get-component-name-from-path "$1")
+  extension=$(_bash-it-get-component-type-from-path "$1")
+  BASH_IT_LOG_PREFIX="$extension: $filename: "
+}
 
 if [ "$1" != "skip" ] && [ -d "./enabled" ]; then
   _bash_it_config_type=""
   if [[ "${1}" =~ ^(alias|completion|plugin)$ ]]; then
     _bash_it_config_type=$1
+    _log_debug "Loading enabled $1 components..."
+  else
+    _log_debug "Loading all enabled components..."
   fi
   for _bash_it_config_file in $(sort <(compgen -G "./enabled/*${_bash_it_config_type}.bash")); do
     if [ -e "${_bash_it_config_file}" ]; then
+      _set-prefix-based-on-path "${_bash_it_config_file}"
       # shellcheck source=/dev/null
       source $_bash_it_config_file
```

Comment:
I would add another `_log_debug` statement here to show which component is loaded at the moment. This will introduce a lot more logging, but might be helpful in some cases.

Reply:
done